association-frontend/components/section-social-media/section-social-media.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+
+import DiscordIcon from "../icons/discord";
+import FacebookIcon from "../icons/facebook";
+import GitHubIcon from "../icons/github";
+import InstagramIcon from "../icons/instagram";
+import LinkedInIcon from "../icons/linkedin";
+import MailIcon from "../icons/mail";
+import TwitterIcon from "../icons/twitter";
+import SectionItem from "../section-item/section-item";
+import SocialCard from "../social-card/social-card";
+
+const SectionSocialMedia = () => {
+  const icons = [
+    <DiscordIcon />,
+    <FacebookIcon />,
+    <GitHubIcon />,
+    <InstagramIcon />,
+    <LinkedInIcon />,
+    <TwitterIcon />,
+    <MailIcon />,
+  ];
+  return (
```

Comment:
```suggestion
import SocialCard from "../social-card/social-card";

const icons = [
  <DiscordIcon />,
  <FacebookIcon />,
  <GitHubIcon />,
  <InstagramIcon />,
  <LinkedInIcon />,
  <TwitterIcon />,
  <MailIcon />,
];

const SectionSocialMedia = () => {
  return (
```

we don't need to define the array every time we render the component